### PR TITLE
Twilio binding metadata

### PR DESCRIPTION
**Pull request by maiqbal11**

### CODE REVIEW

**Comment by maiqbal11** [comment]
fixes #776

**Comment by maiqbal11** [comment]
fixes #744

File: Functions.Templates/Documentation/twilioSmsOut.md [hunk 1]
```diff
@@ -1 +1 @@
-This feature is experimental. Docs will be coming soon!
\ No newline at end of file
+[Click Here for Detailed Documentation](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-twilio)
```
[the remark this answers is not in the document]
Comment (maiqbal11) [reply Functions.Templates/Documentation/twilioSmsOut.md]: Added documentation that will be displayed in portal

File: Functions.Templates/Documentation/twilioSmsOut.md [hunk 2]
```diff
@@ -1 +1,163 @@
...
+- `from` : This value is set to the phone number that the SMS text is sent from
+- `body` : This value can be used to hard code the SMS text message if you don't need to set it dynamically in the code for your function. 
+
+## Example - Functions 2.x
+
+See the language-specific example:
+
+* [2.x C# script (.csx)](#2x-c-script-example)
+* [2.x JavaScript](#2x-javascript-example)
+
```
Comment (soninaren) [inline Functions.Templates/Documentation/twilioSmsOut.md]: Nit: Don't need "2.x" explicitly
You can do * [C# script (.csx)](#c-script-example)

**Pull request merged**